.github/workflows/playwright.yaml

```diff
@@ -0,0 +1,63 @@
+name: Run Playwright tests
```

Comment:
I made a whole new workflow since its conceivable that you'd want to run Playwright tests at a different frequency than unit tests (we certainly did so at Khan Academy)

Reply:
That makes sense... I know that Andrew mentioned that the most efficient way of working with Playwright and Python was to use pytest-playwright... This is why there were deps in in requirements-dev for them and I was going to put everything in one test..


That said I do like the separation of concerns idea mentally and definitely support the idea of putting the playwright tests in their own file.